spec/plaintime.html

```diff
@@ -963,5 +940,31 @@ <h1>RoundTime ( _hour_, _minute_, _second_, _millisecond_, _microsecond_, _nanos
         1. Return ! BalanceTime(_hour_, _minute_, _second_, _millisecond_, _microsecond_, _result_).
       </emu-alg>
     </emu-clause>
+    <emu-clause id="sec-temporal-differencetemporalplaintime" aoid="DifferenceTemporalPlainTime">
+      <h1>
+        DifferenceTemporalPlainTime (
+          _direction_: 1 or -1,
+          _temporalTime_: a Temporal.PlainTime,
+          _other_: an ECMAScript language value,
+          _options_: an Object or *undefined*,
```

Comment:
```suggestion
          _options_: an ECMAScript language value,
```

Reply:
Why should we annotate options as an ECMAScript language value for some type but not the other types?